BAZProject/presentation/viewmodels/MovieViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MovieViewModel.swift
+//  BAZProject
+//
+//  Created by Ivan Tecpanecatl Martinez on 16/04/23.
+//
+
+import Foundation
+
+class MovieViewModel {
+    
+    let movie: Movie
+    
+    var characters: [Actor] = []
+    var onCharactersUpdate: (() -> Void)?
```

Comment:
typealias UpdateHandler = () -> Void
```suggestion
    var onCharactersUpdate: UpdateHandler?
```